python/cudf_polars/cudf_polars/dsl/ir.py

```diff
@@ -1836,7 +1866,9 @@ class Predicate:
 
         def __init__(self, predicate: expr.Expr):
             self.predicate = predicate
-            ast_result = to_ast(predicate)
+            ast_result = to_ast(
```

Comment:
See https://github.com/rapidsai/cudf/pull/20291/files#r2437563679 for context, but this one is for `to_ast`. In this `Predicate.__init__` we again don't have a stream at hand that we can use.